src/Workspaces/Core/Portable/FindSymbols/FindReferences/DependentTypeFinder_Remote.cs

```diff
@@ -25,16 +25,18 @@ internal static partial class DependentTypeFinder
         {
             using (Logger.LogBlock(functionId, cancellationToken))
             {
+                var project = solution.GetOriginatingProject(type);
                 var client = await RemoteHostClient.TryGetClientAsync(solution.Workspace, cancellationToken).ConfigureAwait(false);
-                if (client != null)
+                if (client != null &&
+                    project != null)
```

Comment:
> project != null [](start = 19, length = 16)

can return null immediately if project is null w/o fetching the client

Reply:
true.